ds3-sdk/src/main/java/com/spectralogic/ds3client/helpers/JobPartTracker.java

```diff
@@ -50,4 +50,16 @@ public JobPartTracker attachObjectCompletedListener(final ObjectCompletedListene
         }
         return this;
     }
+
+    public void removeDataTransferredListener(final DataTransferredListener listener) {
+        for (final ObjectPartTracker tracker : this.trackers.values()) {
```

Comment:
A job can have many object part trackers.  The registration code does almost the same thing.